controllers/datadogagent/component/agent/default.go

```diff
@@ -283,3 +307,29 @@ func volumeMountsForSystemProbe() []corev1.VolumeMount {
 		component.GetVolumeMountForAuth(),
 	}
 }
+
+// GetDefaultAgentClusterRolePolicyRules returns the default policy rules for
+// the Agent
+func GetDefaultAgentClusterRolePolicyRules() []rbacv1.PolicyRule {
```

Comment:
should we put this in `feature/enabledefault/rbac.go` instead?

Reply:
Yes, let's do that :+1: I forgot that we already had an `rbac.go` in other features.